SRM 325/RGBStreet.cpp

```diff
@@ -0,0 +1,196 @@
+#include <cstdio>
+#include <cmath>
+#include <cstring>
+#include <ctime>
+#include <iostream>
+#include <algorithm>
+#include <set>
+#include <vector>
+#include <sstream>
+#include <typeinfo>
+#include <fstream>
+
+using namespace std;
+
+int dp[100];
+
+class RGBStreet {
+    public:
+
+    int cvt_strto_int(string s) {
```

Comment:
Write complete - convertStringToInt. Also, either use camel case or snake case but not both.